src/test/java/api/support/builders/RequestBuilder.java

```diff
@@ -646,4 +700,22 @@ public PatronSummary(String lastName, String firstName, String middleName, Strin
       this.barcode = barcode;
     }
   }
+
+
+  public static class Tags {
```

Comment:
This class seems to only provide a list of strings property. Why was this introduced over representing the tags as a List<String> or Set<String>?

Reply:
Sorry MarkI didn't catch. What do you mean "Why was this introduced over representing the tags as a List or Set?"?

Reply:
To me, `tags` in the context of this builder are just a set of strings.

What I'm trying to understand, is why introduce a separate class, that only has a single field for the list, rather than just using a list directly. What benefit does the class give?

I'll make a suggestion in the other part of the builder to demonstrate what I mean.

I'm aware I missed this when reviewing the circulation storage change.

Reply:
I see. I just took the Tags model as is from mod-users. I agree with you that Tags class is unnecessary but i spoke to other guys (much experienced then me). 
This schema is common for all modules and UI component will be able to search many Entities by tags
```  
 "tags": { 
      "type": "object",
      "description": "Tags",
      "$ref": "raml-util/schemas/tags.schema"
    }
```

Reply:
@Anatolii-Starkov I'm not in any way proposing changing the structure of tags in the API.

I'm *only* suggesting that within the builder (used by tests only) there isn't a need for an additional class. The mapping to/from the correct structure can be done just as easily without it.

Does that make sense? If not, I can push a branch describing what I mean.

Reply:
I don't think I'm managing to convey my thoughts around this sufficiently well for it to be actionable, so I am closing this topic :-/